### ISSUE TITLE
Doesn't work with Photoshop CC 2014

### ISSUE THREAD

**Issue by keano12** [kieranblack/velositey#11]
In fact CC 2014 doesn't even have Panel folder.

**Comment by kieranblack** [comment]
hey @keano12 the reason for this is because Adobe has deprecated any extension using Flash as a platform (anything made using Adobe Configurator 4). We are currently working on an HTML-based extension using Eclipse and Extension Builder 3. So stay tuned

**Comment by keano12** [comment]
Great thanks

**Comment by JetPack81** [comment]
awesome! I really want to use this app

**Comment by beninada** [comment]
Waiting on this change! I'm sure many others are as well.

**Comment by keano12** [comment]
Yup it's only when you don't have it you realize how much you miss it

**Comment by kieranblack** [comment]
Hey guys, a bit of an update, we have done two out of the three tabs that are in Velositey. We will be releasing them as three separate panels with a whole bunch of new features for each one. They are all total rewrites and that's why it is taking so long. As a sneak peek, here's the checker/pre-flight panel (please ignore the ugly UI ;) haha)
![image](https://cloud.githubusercontent.com/assets/3181573/3563450/79a40548-0a36-11e4-88c4-c8fd2fbe0090.jpg)

**Comment by stefenphelps** [comment]
any new updates @kieranpblack ? :-)

**Comment by harapeko** [comment]
I am looking forward to updates
✌('ω'✌ )三✌('ω')✌三( ✌'ω')✌

**Comment by TShelton41** [comment]
I am too. I would like to use this for some nice projects coming up.

**Comment by kieranblack** [comment]
Hey guys I know I keep teasing you but it is on it's way, we are just finalising our new website so we can launch v2. Sorry it's taking so long but we are doing this in our spare time as we have 9-5 jobs.

On another note, I have opened up a new repo here https://github.com/kieranpblack/photoshopScripts if any of you are interested. It will contain some functions that I use quite frequently. I will be closing this repo once we have launched v2 of Velositey

@stefenphelps @harapeko @TShelton41 @keano12 @beninada @JetPack81

**Comment by anoldasgupta** [comment]
Hi kieranpblack, 

I just download the velositey-version-2 but its not working like before, i can't find the Styling & Tools panels inside the velositey plugin, i had attached a screen shot of the plugin for your reference, am i missing any features in it according to you...?

![velositey](https://cloud.githubusercontent.com/assets/6092772/4830519/59052e36-5f8e-11e4-87b4-fc3ab13c0676.PNG)

Can you please guide me what might be the issue. Although i am working on latest Photoshop CC 2014 subscrition version.

Thanks in advance, will be waiting for your reply.

Regards
Anol

**Comment by kieranblack** [comment]
Hey Anol,

There's nothing wrong with your version. We will be releasing separate tool panels with these features as a lot of them have to be written from scratch. The next panel will hopefully have all the features that you require. If there are specific tools that you want, add me on Twitter @kieranpblack and we can talk there

**Comment by ctoveloz** [comment]
Anyone have any idea what causes this when I install the velositey v2 ?
I have Adobe Photoshop CC x64
![velociteyy](https://cloud.githubusercontent.com/assets/2256755/4964288/e35cd73a-6733-11e4-8d4e-b44ad87fe854.jpg)

The error is:
This extension can not be installed. The Photoshop64 version is necessary for inclusion in the series between 15.0 and 15.9

**Comment by TShelton41** [comment]
I have downloaded v.2 and moved the folder to my CC 2014 panels folder but I get nothing. Does this still not work? Or do I need to do something else?

**Comment by swinterroth** [comment]
Hi.  I just figured out how to install Velositey on Photoshop CC 2014 (Mac)
  You need to install the CC Extension Manager program from the CC toolbar
icon.   Open it up, and select Photoshop CC 2014 from the left menu.  Click
install extension and upload the Velositey zxp file from the zip.   Close
Photoshop CC 14 and reopen.  Now in the extension menu.

On Sun, Nov 9, 2014 at 7:51 AM, Todd Shelton notifications@github.com
wrote:

> I have downloaded v.2 and moved the folder to my CC 2014 panels folder but
> I get nothing. Does this still not work? Or do I need to do something else?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kieranpblack/velositey/issues/11#issuecomment-62304032
> .

## 

Scott Winterroth
847.553.6020

**Comment by TShelton41** [comment]
@swinterroth Thank you for sharing. It works great.

**Comment by Gonzalo2683** [comment]
Where is the zxp, i downloaded the zip file and the zxp is not here.

**Comment by pfhenley86** [comment]
I'm with Gonzalo2683. I don't see a zxp file to upload and I put it in the correct folder but it is not showing on PS CC 2014 on windows 8

**Comment by swinterroth** [comment]
You need to use the Adobe Extension manager in CC.  Forget the folder
stuff. That's old school.

On Mon, Nov 24, 2014 at 8:32 AM, pfhenley86 notifications@github.com
wrote:

> I'm with Gonzalo2683. I don't see a zxp file to upload and I put it in the
> correct folder but it is not showing on PS CC 2014 on windows 8
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kieranpblack/velositey/issues/11#issuecomment-64201213
> .

## 

Scott Winterroth
847.553.6020

**Comment by pfhenley86** [comment]
OK well the directions are old school. How do I do it via the Extension manager? My cc download did not have a zxp (or whatever the extension file name is) in the download.

**Comment by swinterroth** [comment]
Not sure about the zip file... I would contact the developer on that one.

On Tue, Nov 25, 2014 at 8:36 AM, pfhenley86 notifications@github.com
wrote:

> OK well the directions are old school. How do I do it via the Extension
> manager? My cc download did not have a zxp (or whatever the extension file
> name is) in the download.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/kieranpblack/velositey/issues/11#issuecomment-64407917
> .

## 

Scott Winterroth
847.553.6020

**Comment by pfhenley86** [comment]
Yeah I sent them an email like a week ago and not one responded.

**Comment by wesleycovre** [comment]
Same problem as you, @ctoveloz.

**Comment by wesleycovre** [comment]
Same problem as yours, @ctoveloz.

**Comment by SKempin** [comment]
@pfhenley86  I've found the zxp file - it seems there are two download versions on the Velositey site. Try going to the linked downloads from the below URL. You'll see the zxp file in the expanded zip file. I'm trying to install with Photoshop CC 2015, but this doesn't appear to work.

http://dandkagency.com/extensions/velositey-v2/

**Comment by keano12** [comment]
I get this error for PS C6

```
 On Friday, July 24, 2015 10:29 AM, SKempin <notifications@github.com> wrote:
```

 @pfhenley86 I've found the zxp file - it seems there are two download versions on the Velositey site. Try going to the linked downloads from the below URL. You'll see the zxp file in the expanded zip file. I'm trying to install with Photoshop CC 2015, but this doesn't appear to work.http://dandkagency.com/extensions/velositey-v2/—

**Comment by usarora** [comment]
Any hope to get this working for Photoshop CC 2015?

**Comment by wilixx** [comment]
Any hope to get this working for Photoshop CC 2015?23333

**Comment by mmameen89** [comment]
Any hope to get this working for Photoshop CC 2015?

**Comment by handletastic** [comment]
What's the Photoshop CC 2015 workaround?

**Comment by vanveo** [comment]
I can not find The Panels folder in PTS CC 2015, so what will I do now?

**Comment by alament** [comment]
See my comment here for a workaround for CC 2015: https://github.com/kieranpblack/velositey/issues/13#issuecomment-234801752

**Comment by dinamicostudio** [comment]
It can't work my photoshop cc 2017 please help me ...
